cwms_radar_api/src/test/java/cwms/radar/api/LocationControllerTestIT.java

```diff
@@ -0,0 +1,147 @@
+import fixtures.TestAccounts.KeyUser;
+import io.restassured.RestAssured;
+import io.restassured.path.json.config.JsonPathConfig;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import javax.servlet.http.HttpServletResponse;
+
+import static cwms.radar.api.Controllers.CASCADE_DELETE;
+import static cwms.radar.api.Controllers.OFFICE;
+import static cwms.radar.data.dao.JsonRatingUtilsTest.loadResourceAsString;
+import static io.restassured.RestAssured.given;
+import static io.restassured.config.JsonConfig.jsonConfig;
+import static org.hamcrest.Matchers.is;
+
+@Tag("integration")
+public class LocationControllerTestIT extends DataApiTestIT {
+
+    @Test
+    public void test_location_create_get_delete() throws Exception {
+        String officeId = "SPK";
+        String json = loadResourceAsString("cwms/radar/api/location_create.json");
+        Location location = new Location.Builder(LocationController.deserializeLocation(json, Formats.JSON))
+                .withOfficeId(officeId)
+                .withName(getClass().getSimpleName())
+                .build();
+        String serializedLocation = JsonV1.buildObjectMapper().writeValueAsString(location);
+
+        KeyUser user = KeyUser.SPK_NORMAL;
+        // create location
+        given()
+                .log().everything(true)
+                .accept(Formats.JSON)
+                .contentType(Formats.JSON)
+                .body(serializedLocation)
+                .header("Authorization", user.toHeaderValue())
+                .when()
+                .redirects().follow(true)
+                .redirects().max(3)
+                .post("/locations")
+                .then()
+                .log().body().log().everything(true)
+                .assertThat()
+                .statusCode(is(HttpServletResponse.SC_ACCEPTED));
+        //Create associated time series so delete fails without cascade
+        try {
+            createTimeseries(officeId, location.getName() + ".Flow.Inst.~1Hour.0.radar-test");
+        } catch (Exception ex) {
+
+        }
+
+        // get it back
+        given()
+                .config(RestAssured.config().jsonConfig(jsonConfig().numberReturnType(JsonPathConfig.NumberReturnType.DOUBLE)))
+                .log().everything(true)
+                .accept(Formats.JSON)
+                .header("Authorization", user.toHeaderValue())
+                .queryParam("office", officeId)
+                .when()
+                .redirects().follow(true)
+                .redirects().max(3)
+                .get("/locations/" + location.getName())
+                .then()
+                .log().body().log().everything(true)
+                .assertThat()
+                .statusCode(is(HttpServletResponse.SC_OK));
+
+        // delete without cascade should fail
+        given()
+                .config(RestAssured.config().jsonConfig(jsonConfig().numberReturnType(JsonPathConfig.NumberReturnType.DOUBLE)))
```

Comment:
This should be the default, was it giving you trouble without it?

Reply:
yeah worked without it, I just copy pasted from elsewhere